std/cxx/std/Array.hx

```diff
@@ -17,4 +17,12 @@ extern class Array<T, @:const Count: Int>  {
 	public function maxSize(): cxx.num.SizeT;
 
 	public function fill(value: T): Void;
+
+	@:arrayAccess
+        @:nativeFunctionCode("((*{this})[{arg0}])")
+	public function get(index: Int): T;
+
+	@:arrayAccess
+        @:nativeFunctionCode("(*{this})[{arg0}] = {arg1}")
```

Comment:
I propose to map this to at: https://en.cppreference.com/w/cpp/container/array/at